hummingbird/ml/supported.py

```diff
@@ -119,10 +119,16 @@ def _build_sklearn_operator_list():
             StandardScaler,
         )
 
+        try:
+            from sklearn.preprocessing import Imputer
+        except ImportError:
+            # Imputer was deprecate in sklearn >= 0.22
```

Comment:
We could also do something like 
```
import sklearn
if sklearn.__version__ < 0.22:
....
```
but maybe this way is simpler :)

Reply:
Yes. That is what I was doing at the beginning but then realized that this is way simpler! :)